Python.gitignore

```diff
@@ -76,6 +76,9 @@ target/
 # celery beat schedule file
 celerybeat-schedule
 
+#Sage parsed files
```

Comment:
SageMath seems to be the latest name here (as per Wikipedia at least). Would that make this a bit clearer?